Client/Application/AppDelegate.swift

```diff
@@ -127,11 +128,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         navigationController.delegate = self
         navigationController.isNavigationBarHidden = true
 
-        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
-            rootViewController = NotificationRootViewController(rootViewController: navigationController)
-        } else {
+//        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
+//            rootViewController = NotificationRootViewController(rootViewController: navigationController)
+//        } else {
             rootViewController = navigationController
-        }
+//        }
```

Comment:
This was an old feature that never made it to release. It was kind of buggy so it might be worth either removing entirely or making the deep links worth with it. Essentially it replaces the UINavigationController we setup with a subclass that shows a notification view in place of the status bar. I think it's fine to keep commented for now but I would just add a comment.